packages/dialtone-tokens/tokens/semantic/dp/default.json

```diff
@@ -1119,6 +1119,300 @@
+          "type": "color"
+        },
+        "10-selected": {
+          "value": "{color.teal.900}",
+          "type": "color"
+        },
+        "01": {
+          "value": "{color.purple.450}",
+          "type": "color"
+        },
+        "01-hover": {
+          "value": "{color.purple.500}",
+          "type": "color"
+        },
+        "01-selected": {
+          "value": "{color.purple.600}",
+          "type": "color"
+        },
+        "02": {
+          "value": "{color.magenta.300}",
+          "type": "color"
+        },
+        "02-hover": {
+          "value": "{color.magenta.400}",
+          "type": "color"
+        },
+        "02-selected": {
+          "value": "{color.magenta.600}",
+          "type": "color"
+        },
+        "03": {
+          "value": "{color.blue.400}",
+          "type": "color"
+        },
+        "03-hover": {
+          "value": "{color.blue.450}",
+          "type": "color"
+        },
+        "03-selected": {
+          "value": "{color.blue.600}",
+          "type": "color"
+        },
+        "04": {
+          "value": "{color.gold.400}",
+          "type": "color"
+        },
+        "04-hover": {
+          "value": "{color.gold.500}",
+          "type": "color"
+        },
+        "04-selected": {
+          "value": "{color.gold.700}",
+          "type": "color"
+        },
+        "05": {
+          "value": "{color.green.425}",
+          "type": "color"
+        },
+        "05-hover": {
+          "value": "{color.green.475}",
+          "type": "color"
+        },
+        "05-selected": {
+          "value": "{color.green.600}",
+          "type": "color"
+        },
+        "06": {
+          "value": "{color.berry.300}",
+          "type": "color"
+        },
+        "06-hover": {
+          "value": "{color.berry.400}",
+          "type": "color"
+        },
+        "06-selected": {
+          "value": "{color.berry.600}",
+          "type": "color"
+        },
+        "07": {
+          "value": "{color.coral.400}",
+          "type": "color"
+        },
+        "07-hover": {
+          "value": "{color.coral.500}",
+          "type": "color"
+        },
+        "07-selected": {
+          "value": "{color.coral.900}",
+          "type": "color"
+        },
+        "08": {
+          "value": "{color.olive.500}",
+          "type": "color"
+        },
+        "08-hover": {
+          "value": "{color.olive.600}",
+          "type": "color"
+        },
+        "08-selected": {
+          "value": "{color.olive.800}",
+          "type": "color"
+        },
+        "09": {
+          "value": "{color.red.350}",
+          "type": "color"
+        },
+        "09-hover": {
+          "value": "{color.red.400}",
+          "type": "color"
+        },
+        "09-selected": {
+          "value": "{color.red.600}",
+          "type": "color"
+        }
+      },
+      "neutral": {
+        "value": "{color.tan.300}",
+        "type": "color"
+      },
```

Comment:
For each chart color, add `description` field with a one-liner describing the chart color, much like you see on https://github.com/dialpad/dialtone/blob/ad3665aeef763f37647fada6dbb31c29bf19c8ad/packages/dialtone-tokens/tokens/semantic/dp/default.json

This `description` field is what you see on https://dialtone.dialpad.com/tokens/?search=critical

Note of course that not all design tokens have a `description`. It's a judgement call. They don't need to be super wordy, just simple and straight-forward. 

<img width="929" height="250" alt="Image" src="https://github.com/user-attachments/assets/e3009774-8668-4bee-badf-91e5d2f7b3c6" />